packages/schema/src/config/build.ts

```diff
@@ -38,6 +39,20 @@ export default defineUntypedSchema({
     },
   },
 
+  /**
+   * Log level when building logs.
+   *
+   * Defaults to 'quiet' when running in CI or when a TTY is not available.
+   * This option is then used as 'silent' in Vite and 'none' in Webpack
+   *
+   * @type {'quiet' | typeof import('../src/types/global/vite').ViteLogLevel | typeof import('../src/types/global/webpack').WebpackStats}
```

Comment:
I think we should create our own list of possible values rather than allowing _any_ possible vite/webpack value. We could support a subset of the vite values as that's our default setup, and map them to webpack values:

So, to start with:

- `silent` -> silent/none
- `info` -> info/normal

Reply:
So basically to create an enum of NuxtLogs values like silent|info for now and do not accept any other?

If we do this, users would have to access vite/webpack configs directly from nuxt.config.ts. to change it to any other vite or webpack log option.

Is that correct?

Reply:
I wouldn't create an enum; just use string literals. But yes, that's right.

Reply:
I love the idea for having built-in nuxt levels and then map them (otherwise we can never find single convention 😆). If i may suggest, we shall add at least a 3rd `verbose` variant for CLI compat (silent: nothing, info: default normal, vebose: all logs)

Reply:
Agreed 👍

Reply:
Not sure about verbose value 
https://vitejs.dev/config/shared-options.html#loglevel
The info is a default one
'info' | 'warn' | 'error' | 'silent'
So Nuxt log level info | verbose | silent would only work for Vite info and silent

Reply:
Yes, so we basically **map** this values. In this case for vite, both info and verbose map to `info` for vite as it has nothing more.

Reply:
I missed your message somehow @pi0. Sorry for that. Adding them right now so that we can proceed :)